### PR TITLE
Add battery usage setting to VPN

### PR DESCRIPTION
**Pull request by aitorvs**
Task/Issue URL: https://app.asana.com/0/488551667048375/1205997010028871/f

### Description
See Asana task

### Steps to test this PR
_Test_
- [x] install from this branch
- [x] enable NetP
- [x] go to NetP management screen -> VPN settings?
- [x] tap on "Allow Unrestricted app battery usage"
- [x] verify user is taken to DDG app info
- [x] go back to the app
- [x] verify the title/byline didn't change
- [x] tap on "Allow Unrestricted app battery usage" -> battery and set DDG to "unrestricted"
- [x] go back to DDG app
- [x] verify the title changed to "app battery usage is unrestricted"
- [x] Tap on item and set battery to "Optimized"
- [x] go back to DDG app
- [x] verify the title changed to "allow unrestricted app battery usage"

### CODE REVIEW

File: network-protection/network-protection-impl/src/main/res/values/donottranslate.xml [hunk 1]
```diff
@@ -126,6 +126,10 @@
     <!-- VPN settings -->
     <string name="netpVpnSettingKillSwitch">Always On</string>
     <string name="netpVpnSettingAlwaysOn">Always On</string>
+    <string name="netpManageRecentAppsProtectionAllowUnrestrictedBattTitle">Allow Unrestricted App Battery Usage</string>
+    <string name="netpManageRecentAppsProtectionAllowUnrestrictedBattByline">In App Battery Settings, allow Unrestricted usage to prevent Android from disabling App Tracking Protection in the background</string>
```
Comment (karlenDimla) [inline network-protection/network-protection-impl/src/main/res/values/donottranslate.xml:130]: I think this should be Network Protection than App Tracking Protection

File: network-protection/network-protection-impl/src/main/res/values/donottranslate.xml [hunk 2]
```diff
@@ -126,6 +126,10 @@
...
+    <string name="netpManageRecentAppsProtectionUnrestrictedBattTitle">App Battery Usage Is Unrestricted</string>
+    <string name="netpManageRecentAppsProtectionUnrestrictedBattByline">In App Battery Settings, this prevents Android from disabling App Tracking Protection in the background</string>
```
Comment (karlenDimla) [inline network-protection/network-protection-impl/src/main/res/values/donottranslate.xml:132]: ditto

**Review by karlenDimla — CHANGES_REQUESTED** [review]
We need to update the copy and not mention App Tracking Protection on the Netp screen

File: network-protection/network-protection-impl/src/main/res/values/donottranslate.xml (hunk 2 above)
Comment (aitorvs) [reply network-protection/network-protection-impl/src/main/res/values/donottranslate.xml]: done

File: network-protection/network-protection-impl/src/main/res/values/donottranslate.xml (hunk 1 above)
Comment (aitorvs) [reply network-protection/network-protection-impl/src/main/res/values/donottranslate.xml]: done

**Review by karlenDimla — APPROVED** [review]

**Pull request merged**